src/main/java/spoon/pattern/PatternBuilder.java

```diff
@@ -396,7 +398,12 @@ PatternBuilder configureLocalParameters(Consumer<ParametersBuilder> parametersBu
 		return this;
 	}
 	/**
-	 * adds all standard Template parameters based on {@link TemplateParameter} and {@link Parameter} annotation
+	 *
+	 * 	All variables defined out of scope of the model (eg body of the method)
+	 * 	are considered as template parameters
```

Comment:
It is not true for this method. The method `configureTemplateParameters` creates parameters in same way like legacy Templates did it. The comment you have entered here would be valid for `createPatternParameters` method.
So I suggest to rollback this javadoc to origin version.